DSCResources/MSFT_xDnsRecord/MSFT_xDnsRecord.schema.mof

```diff
@@ -0,0 +1,9 @@
+[ClassVersion("1.0.0.0"), FriendlyName("xDnsRecord")]
+class MSFT_xDnsRecord : OMI_BaseResource
+{
+    [Key] string Name;
+    [Key] string Zone;
+    [Required,ValueMap{"A-record", "C-name"},Values{"A-record", "C-name"}] string Type;

```

Comment:
Should it be CName and ARecord instead? That's what I see online and in powershell cmdlets